tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -817,8 +817,11 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         return this._dashboardData
             .map((d) => d.plugin)
             .filter((dashboardName) => {
-                return this._pluginsListing[dashboardName] &&
-                    this._pluginsListing[dashboardName].enabled;
+                const maybeMetadata = this._pluginsListing[dashboardName];
+                if (typeof maybeMetadata === 'boolean') {
+                  return maybeMetadata;
+                }
+                return this._pluginsListing[dashboardName].enabled;
```

Comment:
nit: Replace `this._pluginsListing[dashboardName]` with `maybeMetadata`?

A brief comment about the reason for this business (namely, API compat)
would be reasonable imho, but I’ll leave that up to you.